Source/Core/Core/NetPlayProto.h

```diff
@@ -104,6 +108,20 @@ enum
   CON_ERR_VERSION_MISMATCH = 3
 };
 
+enum
+{
+  SYNC_SAVE_DATA_NOTIFY = 0,
+  SYNC_SAVE_DATA_SUCCESS = 1,
+  SYNC_SAVE_DATA_FAILURE = 2,
+  SYNC_SAVE_DATA_RAW = 3,
+  SYNC_SAVE_DATA_GCI = 4,
+  SYNC_SAVE_DATA_WII = 5,
+  SYNC_SAVE_DATA_SD = 6
+};
+
+const u32 NETPLAY_LZO_IN_LEN = 1024 * 64;
```

Comment:
These can be `constexpr`

Reply:
What about the one below it?

Reply:
Yeah, that one can be made constexpr as well (which is what I meant by 'these', but I should have been more specific)